kotlin/internal/jvm/impl.bzl

```diff
@@ -300,7 +300,7 @@ def kt_jvm_junit_test_impl(ctx):
             direct = ctx.files._java_runtime,
         ),
         # adds common test variables, including TEST_WORKSPACE.
-        testing.TestEnvironment({}),
+        testing.TestEnvironment(environment = ctx.attr.env),
```

Comment:
I think this change is only compatible with Bazel 6.0.  Earlier versions of Bazel fail for me with
```
	File "external/io_bazel_rules_kotlin/kotlin/internal/jvm/impl.bzl", line 306, column 32, in kt_jvm_junit_test_impl
		testing.TestEnvironment(environment = ctx.attr.env),
Error in TestEnvironment: TestEnvironment() got named argument for positional-only parameter 'environment'
```